CONTRIBUTING.md

```diff
@@ -18,8 +18,7 @@ standard library, so that users can experiment with them before they are added t
 standard library. Such features should ideally already be specified in a PEP or draft
 PEP.
 
-`typing_extensions` still supports all Python versions supported by `typing`, down to
-Python 2.7 and 3.4. However, it is OK to omit support for Python versions that have
+`typing_extensions` supports Python 3.6+. However, it is OK to omit support for Python versions that have
 reached end of life if doing so is too difficult or otherwise does not make sense. For
 example, `typing_extensions.AsyncGenerator` only exists on Python 3.6 and higher,
```

Comment:
Updated to "For example, `typing_extensions.AsyncGenerator` was only supported in Python 3.6 and newer, because async generators were not part of the language before then."

A

Reply:
You could also phrase in the negative -- e.g. "For example, `typing_extensions.AsyncGenerator` was not supported in Python versions before 3.6, as async generators were not part of the language."